code/packages/web/src/pages/ForgotPasswordPage.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const ForgotPasswordPage = () => {
+  return (
+    <div>
+      <h2>Forgot Password?</h2>
+      <form>
```

Comment:
You should also add a form submit function. Basically form submission should trigger onSubmit which will need to be a function. Leave the function empty for now. Do that for ForgotPasswordPage, LoginPage and SignupPage